scheduler/node/node.go

```diff
@@ -0,0 +1,86 @@
+package node
```

Comment:
I believe it has to be a package, otherwise you always end up with import cycle.
It's used by cluster/swarm and scheduler. At least I didn't managed to do without it.